### PR TITLE
Comments addressed

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
#### Descriptions:
1. Used classes instead of style
2. The logo wasn't aligned with the text below
Before:
image
image

After:
Screenshot 2023-12-13 at 10 19 20 AM

### CODE REVIEW

File: widget/view.html [hunk 1]
```diff
@@ -3,62 +3,77 @@
...
+                      <div data-ng-if="item.icon_small">
+                        <img class="connector-logo-dimensions" data-ng-src="{{item.icon_small}}" alt="Base64 Image">
+                      </div>
+                      <div class="margin-left-18">
+                        <div class="text-overflow" title="{{item.label}}">{{item.label}}</div>
+                        <div class="connector-card-secondary-header-text"><span>{{item.configuration.length}}
+                            {{(item.configuration.length === 1)?'Configuration':'Configurations'}} Monitored</span></div>
+                      </div>
+                    </div>
+                    <div class="margin-left-auto;">
```
Comment (cs-shabaz-salmani) [inline widget/view.html]: improper class

File: widget/view.html [hunk 2]
```diff
@@ -3,62 +3,77 @@
...
                   <div class="connector-config-wrapper">
                     <div data-ng-hide="collapse" class="col-md-12 connector-config">
                       <div class="row margin-top-md">
                         <div class="col-md-6">
-                          <span class="pull-left muted font-10px">Configuration Name</span>
+                          <span class=" muted font-10px">Configuration Name</span>
                         </div>
                         <div class="col-md-6">
-                          <div class="pull-right">
+                          <div class="position-absolute right-10">
```
Comment (cs-shabaz-salmani) [inline widget/view.html]: position absolute not required here

File: widget/view.html [hunk 3]
```diff
@@ -3,62 +3,77 @@
...
+                      <div class="config-health-status row"
+                        data-ng-repeat="items in item.configuration track by $index">
                         <div class="col-md-6 padding-left-0 padding-right-sm">
-                          <span class="pull-left connector-config-header-text">{{items.name}} <span data-ng-if="items.agentName"> - {{items.agentName}}</span></span>
+                          <span class="connector-config-header-text">{{items.name}} <span data-ng-if="items.agentName">
+                              - {{items.agentName}}</span></span>
                         </div>
                         <div class="col-md-6 padding-right-0 padding-left-sm">
-                          <div class="pull-right">
+                          <div class="position-absolute config-health-message-position">
```
Comment (cs-shabaz-salmani) [inline widget/view.html]: position absolute not required

Comment (cs-yash-patodkar) [reply widget/view.html]: If position-absolute not used Right doesnt work, to stick it to right I've used position absolute

File: widget/view.html (hunk 2 above)
Comment (cs-yash-patodkar) [reply widget/view.html]: If position-absolute not used Right doesnt work, to stick it to right I've used position absolute

File: widget/view.html (hunk 3 above)
Comment (cs-shabaz-salmani) [reply widget/view.html]: position-absolute is not required here
Use display flex and justify the content to align in start and end

**Comment by cs-shabaz-salmani** [comment]
Looks good

**Pull request merged**